frontend/app/views/spree/shared/_currency_dropdown.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if defined?(should_render_currency_dropdown?) && should_render_currency_dropdown? %>
-  <%= form_tag spree.set_currency_path, method: :get, class: 'px-4 py-3 w-100', id: 'currency-select' do %>
+  <%= form_tag spree.set_currency_path, method: :get, class: 'px-4 py-3 w-100 currency-select' do %>
```

Comment:
Note: currently we don't have `currency-select` class (it's unused)
ID was used here:
https://github.com/spree/spree/pull/10812/files#diff-0edd9601602c7e331cadf47258e4ae67dc022798d4d3fb364ba1be127462cc3cL2
Same with `locale-select` class.

Reply:
@KacperMekarski yes, we had 2 elements with the same ID on a single page so that was a no-go, that's why this change